packages/table/src/headers/columnHeader.tsx

```diff
@@ -36,7 +36,7 @@ export interface ColumnWidths {
     defaultColumnWidth: number;
 }
 
-export interface ColumnHeaderProps extends HeaderProps, ColumnWidths, ColumnIndices {
+export interface ColumnHeaderProps extends React.PropsWithChildren<HeaderProps>, ColumnWidths, ColumnIndices {
```

Comment:
nitpick: Similar comment around potentially writing the `children` field directly.

Reply:
21a62fcc8a12121739b04430f25dc05d1e14da19